modules/migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v530/MigrationDatabaseCreator.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
It is not necessary to update the year. The year means the license first enforced(and has no any meaning)